src/control/lib/control/pool.go

```diff
@@ -1241,6 +1240,10 @@ func GetMaxPoolSize(ctx context.Context, log logging.Logger, rpcClient UnaryInvo
 	}
 
 	hostStorageMap := resp.HostStorage
+	// DAOS-10885: The variable ranksWithoutSmd is used for checking if there is no usable ranks
```

Comment:
This comment confuses me, maybe you could try to make it more clear?

Reply:
- [ ] Makes comment more comprehensible